skel/gauge.properties

```diff
@@ -20,3 +20,9 @@ runner_request_timeout = 30000
 
 # Allow Gauge and its plugin updates to be notified.
 check_updates = true
+
+# Allow Gauge to collect anonymous usage statistics
+gauge_analytics_enabled = true
```

Comment:
The value should be `false` here. The analytics should be turned off by default.